src/app/teamSheets/_TeamSheetListItem/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faEdit, faShareFromSquare } from '@fortawesome/free-solid-svg-icons'
+
+import Link from 'next/link'
+
+import type { TeamSheetWithRelations } from '@types'
+
+interface TeamSheetListItemProps {
+  teamSheet: TeamSheetWithRelations,
+}
+
+const TeamSheetListItem = (props: TeamSheetListItemProps) => {
+  const { teamSheet } = props || {}
+  const { editId, shareId, team, title: teamSheetTitle } = teamSheet
+  const { competitionTeams, key: teamKey, title: teamTitle } = team
+
+  const hasMultipleCompetitionTeams = !!competitionTeams.length
+  const firstCompetitionTeam = competitionTeams[0]
+  const { 
+    competition: {
+      title: competitionTitle,
+      sport: {
+        key: sportKey,
+      },
+    },
+  } = firstCompetitionTeam
+
+  return (
+    <div className="flex border rounded mb-3 p-4 w-full justify-between">
+      <div className="flex flex-col">
+        {teamSheetTitle && (
+          <div className="font-semibold">
+            {teamSheetTitle}
+          </div>
+        )}
+        
+        <div className="flex text-sm text-slate-400">
+          <div>{teamTitle}</div>
+
+          {!hasMultipleCompetitionTeams && (
+            <div>, {competitionTitle}</div>
```

Comment:
i don't think showing this makes sense

there is currently no way of knowing which competition a teamSheet is for, we would have to add a relation for competition directly on teamSheet for that

do we expect a user to pick different players for the same team in different competitions?